### ISSUE TITLE
OwnCloud 8 RainLoop plugin will not let me log in after update

### ISSUE THREAD

**Issue by Luticus** [RainLoop/rainloop-webmail#493]
I originally had RainLoop working with the OwnCloud plugin on OwnCloud 7, after updating to 8 i can no longer log into my email (says:  authentication failure).  Log in still works fine when I go directly to the RainLoop web client, and only fails when I use the OwnCloud plugin.

**Comment by kbridger** [comment]
I'm seeing the same problem.  I now understand that the OC7 -> OC8 upgrade actually resulted in installing a completely new Rainloop instance under the OC apps directory.  I went through the new instance's admin panel to configure things.  I was able to set the OC user's credentials in the OC Personal settings.   Everyhting seems to work until I try to actually log in as a user.  Then auth fails.

My username for Rainloop is not the same as my OC username, so I don't have that option set.

I'm not even sure where the Rainloop logs would be to check why auth is failing.

**Comment by RainLoop** [comment]
Try to check RainLoop's logs.
```
<rainloop-data-folder>/...../logs/
```

Config:
```
<rainloop-data-folder>/...../application.ini
```
```ini
[logs]
enable = On
```

**Comment by kbridger** [comment]
Could you help me find the right folders/config files?

Assume I have owncloud in my root folder (so /var/www/owncloud).

My data folder contains 2 files: EMPTY and VERSION
/var/www/owncloud/apps/rainloop/app/data

I don't see an application.ini at all via find /var/www/owncloud application.ini

**Comment by RainLoop** [comment]
Yes, you may find it (RainLoop data folder) into ownCloud data folder.

**Comment by Luticus** [comment]
mine is in my /var/www/*rainloop*/data/_data_.../_default_/

however my logs work when I log in via rainloop web app, from the owncloud app it doesn't seem to generate logs at all.  All I get is "Authentication Failed".  Again, I only experience this in the owncloud plugin, rainloop itself works fine.  The previous owncloud plugin, before I updated to owncloud 8, also worked fine.  I'll see if I can maybe find something in the owncloud logs.

**Comment by RainLoop** [comment]
> The previous owncloud plugin, before I updated to owncloud 8, also worked fine. 

@Luticus 
Sorry, but new version works only with RainLoop ownCloud package.
Why do you use external RainLoop application?

**Comment by kbridger** [comment]
I've enabled logging after finding the data directory in the owncloud storage, not in /var/www/owncloud

It looks like the login is trying to authenticate me with my domain email, which isn't my username on the IMAP server.  So I cleared the default domain setting, leaving it blank.  Now when I try to login rainloop thinks the login is not a valid email, so doesn't even try to authenticate with the IMAP server.

Maybe this is misconfigured, but how do I get rainloop to try authenticating with just the "username" provided into the "email" field?

**Comment by RainLoop** [comment]
Try to enable "Use short login" setting in domain popup.

![image](https://cloud.githubusercontent.com/assets/5530092/6354925/ba901828-bc63-11e4-8a54-1405e5b315ad.png)

**Comment by kbridger** [comment]
Ah - that did it.  This was because I had the standalone rainloop install in OC7 configured.  When I updated to OC8 and installed the new owncloud rainloop app it already had some of my config, including valid domains.  I forgot to double check the domain settings.

Works great now - thanks!

**Comment by RainLoop** [comment]
You welcome :)

**Comment by Luticus** [comment]
Alright I've figured out what the problem was.  Aparently I thought that the owncloud plug in just used the settings from the regular rainloop.  Now that I've figured out that they are decoupled and I saw the admin panel access in ownclouds settings, everything seems to work fine.  

Well except for contacts, I'm getting an error when i tell it to connect to the database that says "Personal address book driver not allowed".  Let me know if you know a quick fix or if I should open a new issue.

Thanks for all your help :)
